suite-native/module-accounts-import/src/accountsImportSlice.ts

```diff
@@ -0,0 +1,33 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export const actionPrefix = '@accountsImport';
+
+export type AccountsImportState = Record<string, string>;
+
+const initialState: AccountsImportState = {};
```

Comment:
I assume there will be other properties to this reducer so I would create object property accountNames inside this object.

Reply:
Actually I don't expect to be here more than key:value descriptor:accountName because whole account functionality is covered in the common account package in `wallet-core`. This is here only because of "labeling" - web/desktop suite handles this labeling somehow with Dropbox integration, I used this storage. What do you think?

Reply:
Please also create selector for getting account name from state. If we will use selector we can refactor whole structure of this slice state without need to modify single line outside this file so we can solve this issue later.